domains/harshaljariwala.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Hjgaming",
+           "email": "jariwalaharshal80@gmail.com",
+           "discord": "811259286977773599"
+        },
+    
+        "record": {
+            "TXT": "vc-domain-verify=harshaljariwala.is-a.dev,86a3a60852704797b875"
```

Comment:
You can't use vercel yet

Reply:
not went harshaljariwala.is-a.dev i need botstatus.is-a.dev  check that